### ISSUE TITLE
Standard Keyboard

### ISSUE THREAD

**Issue by jimjahan** [Unity-Technologies/EditorVR#151]
No keyboard is shown when I select string fields in inspector. I guess the Standard Keyboard should be shown.
Selecting numeric fields shows numeric keyboard correctly.
I tried by 0.0.4 and Vive.

**Comment by amirebrahimi** [comment]
Yes, this is currently disabled and is a feature that will need to be added.
